Dockerfile

```
@@ -0,0 +1,36 @@
+FROM python:3.8-slim
+RUN apt update && apt install -y wget git libglib2.0-0 libsm6 libxext6 libxrender-dev
+RUN apt clean && rm -rf /var/lib/apt/lists/*
+# RUN useradd -ms /bin/bash stablediff
+# install conda
+RUN wget \
+    https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh \
+    && echo PATH="/root/miniconda3/bin":$PATH >> .bashrc 
+RUN chmod +x /root/miniconda3/bin/conda
+RUN ln -s /root/miniconda3/bin/conda /usr/local/bin/conda
+RUN conda update -y conda
```

The official [continuumio/miniconda3](https://hub.docker.com/r/continuumio/miniconda3) container images would spare a few lines

```suggestion
FROM continuumio/miniconda3
```